src/Handler/SentryHandler.php

```diff
@@ -72,7 +72,10 @@ public function __construct($level = null, bool $bubble = true, array $config =
         $level = Level::fromName($level ?? Level::Debug->getName());
 
         SentrySdk::setCurrentHub(new Hub($client));
-        
+
+        $trace_sample_rate = SentryAdaptor::get_opts()['traces_sample_rate'] ?? null;
+        $config['traces_sample_rate'] = $trace_sample_rate ?: 0.0;
```

Comment:
@clodeindustrie why not do `$config['traces_sample_rate'] = SentryAdaptor::get_opts()['traces_sample_rate'] ?? 0.0;`? Saves you a step